app/helpers/quarter_helper.rb

```diff
@@ -32,7 +32,7 @@ def quarter_string(date)
     year = date.year
     quarter = quarter(date)
 
-    "#{year} Q#{quarter}"
+    "Q#{quarter}-#{year}"
```

Comment:
@harimohanraj89 added the hyphen to the quarter string. Did a quick check, and don't think this breaks any tests.